tools/packaging/packaging.psm1

```diff
@@ -2268,6 +2276,9 @@ function New-ReferenceAssembly
             Copy-Item $refBinPath $RefAssemblyDestinationPath -Force
             Write-Log "Reference assembly '$assemblyName.dll' built and copied to $RefAssemblyDestinationPath"
 
+            Copy-Item $dllXmlDoc $RefAssemblyDestinationPath -Force
+            Write-Log "Xml document '$assemblyName.xml' copied to $RefAssemblyDestinationPath"
```

Comment:
`... make NuGet packages include the XML document files.`
This is not really related to prediction APIs. Could have been a separate PR.

Reply:
Not directly related, but sort of. The script needs to be updated after the changes in API because new attributes are used which need to be scrubbed when generating the reference assembly, so I have to update the packaging scripts. While doing so, I found the XML docs are missing when using our NuGet packages in VS Code or Visual Studio, and thus I fix that too.
Since we need to include part of the fixes in `packaging.psm1` anyways, I prefer to keep all changes in this PR.